tests/test_unsafe.py

```diff
@@ -335,7 +335,7 @@ def test_update_cli(
         git("tag", "v1")
 
     _, retcode = CopierApp.run(
-        ["copier", "copy", "--UNSAFE", str(src), str(dst)],
+        ["copier", "copy", "--trusted", str(src), str(dst)],
```

Comment:
How about parametrizing this test to check that both `--UNSAFE` and `--trusted` both work?

Reply:
I don't think it's needed, because the same test uses `--UNSAFE` a few lines later.

Reply:
Hm, I think it would make the test clearer (one might wonder, is there a particular reason why it's first `--trusted`/`--trust` and later `--UNSAFE`) and possibly add a tiny bit of safety.

Reply:
Done.